packages/schematics/angular/application/files/.browserslistrc.template

```diff
@@ -2,11 +2,17 @@
 # For additional information regarding the format and rule options, please see:
 # https://github.com/browserslist/browserslist#queries
 
+# For the full list of supported browsers by the Angular framework, please see:
+# https://angular.io/guide/browser-support
+
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
-last 2 versions
-Firefox ESR
-not dead
-not IE 9-11 # For IE 9-11 support, remove 'not'.
\ No newline at end of file
+last 1 Chrome version
+last 1 Firefox version
+last 2 Edge major versions
+last 2 Safari major version
+last 2 iOS major versions
+Firefox ESR<% if (legacyBrowsers) { %>
+IE 11<% } else { %>
```

Comment:
I suggest that you change this to include IE9 and 10 for now to unblock this PR. And once we have a decision on IE9/10 and if the decision is positive then you can send a small PR to drop those two browsers from here.

Reply:
Okay, let's do that.